services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/task.py

```diff
@@ -500,14 +504,13 @@ async def setup_scheduler(app: FastAPI):
 
 async def shutdown_scheduler(app: FastAPI):
     scheduler: DynamicSidecarsScheduler = app.state.dynamic_sidecar_scheduler
-    ## FIXME: somehow this does not work
-    # assert isinstance(scheduler, DynamicSidecarsScheduler)  # nosec
+    assert isinstance(scheduler, DynamicSidecarsScheduler)  # nosec
 
     # FIXME: PC->ANE: if not started, should it be shutdown?
```

Comment:
maybe have a  look at this FIXME too?

Reply:
I left it in there for a later review.
The action pointed out is already being handled by the function.